### PR TITLE
change URL of slip-0044

### PR DESCRIPTION
**Pull request by prusnak**

### CODE REVIEW

File: constants.json [hunk 1]
```diff
@@ -52,5 +52,8 @@
   "Electronic Gulden": "0x8000004e",
   "ClubCoin": "0x8000004f",
   "RichCoin": "0x80000050",
-  "Potcoin": "0x80000051"
+  "Potcoin": "0x80000051",
+  "Quarkcoin": "0x80000052",
+  "Terracoin": "0x80000053",
+  "Gridcoin": "0x80000054"
 }
```
Comment (fanatid) [inline constants.json]: no newline?

**Comment by fanatid** [comment]
ACK

File: constants.json (hunk 1 above)
Comment (prusnak) [inline constants.json]: I just run your script on my machine to check whether it still works.

When you look at the diff, you'll see the original file did not contain new line either.

Comment (fanatid) [inline constants.json]: oops, sorry! PR: https://github.com/prusnak/bip44-constants/pull/1 ;)

**Comment by jprichardson** [comment]
Thanks!

**Pull request merged**